sklearn_pandas/categorical_imputer.py

```diff
@@ -46,10 +52,17 @@ class CategoricalImputer(BaseEstimator, TransformerMixin):
 
     """
 
-    def __init__(self, missing_values='NaN', replacement='mode', copy=True):
+    def __init__(
+        self,
+        missing_values='NaN',
+        strategy='mode',
+        replacement='?',
```

Comment:
Could it be `replacement=None` instead?

Reply:
Ok, that seems like a good approach. I'll see if I can add some parameter validation for `strategy` as well.